marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/transforms/mlcp-flow-transform.sjs

```diff
@@ -62,7 +62,7 @@ function transform(content, context = {}) {
 
     if (!flow) {
       datahub.debug.log({message: params, type: 'error'});
-      fn.error(null, "RESTAPI-SRVEXERR", Sequence.from(["DH-FLOWMISSING", "The specified flow " + flowName + " is missing.", content.uri]));
+      fn.error(null, "RESTAPI-SRVEXERR", Sequence.from([404, "DH-FLOWMISSING", "The specified flow " + flowName + " is missing.", content.uri]));
```

Comment:
For these, let's add throwNotFoundWithArray(args) and throwBadRequestWithArray(args) so that when a client wants to return multiple things, it's easy to do so.